src/main/java/sirius/biz/storage/layer3/downlink/ssh/SSHServer.java

```diff
@@ -137,25 +137,34 @@ private boolean authenticate(String username, String password, ServerSession ses
             username = scopeAndUsername.getSecond();
         }
 
-
         boolean locked =
                 UserContext.getCurrentScope().tryAs(MaintenanceInfo.class).map(MaintenanceInfo::isLocked).orElse(false);
         if (locked) {
             return false;
         }
 
         if (StorageUtils.LOG.isFINE()) {
-            StorageUtils.LOG.FINE("Layer 3/FTP: Trying to authenticate user: " + username);
+            StorageUtils.LOG.FINE("Layer 3/SSH: Trying to authenticate user: " + username);
         }
 
         UserInfo authUser = UserContext.get().getUserManager().findUserByCredentials(null, username, password);
+
+        if (authUser == null) {
+            return false;
+        }
+
+        if (!authUser.isSubScopeEnabled(VirtualFileSystem.SUB_SCOPE_VFS)) {
+            StorageUtils.LOG.FINE("Layer 3/SSH: The required sub scope is not enabled for: " + username);
```

Comment:
Maybe log on Info?

Reply:
all other auth relevant messages are FINE as well therefore I used the same level here...